index.html

```diff
@@ -1065,11 +1065,10 @@ <h3>Contexts</h3>
 property be processed using JSON-LD. This is to support processing using plain
 JSON libraries such as those that may be used when the
 <a>verifiable credential</a> is encoded as a JWT.
-JSON-LD aware processors can process the <code>@context</code> using full
-JSON-LD processing as expected. For other processors, the only processing
-necessary is to ensure that the order of the values in <code>@context</code>
-is what is expected for the particular application, but no JSON-LD processing
-of those values is required.
+All libraries or processors MUST ensure that the order of the values in the
+<code>@context</code> <a>property</a> is what is expected for the particular
+application. Libraries or processors that support JSON-LD SHOULD
+process the <code>@context</code> <a>property</a> using full JSON-LD processing.
```

Comment:
```suggestion
process the <code>@context</code> <a>property</a> using full JSON-LD processing as needed.
```

I suggest we add `as needed` at the end there because you don't *need* to do any JSON-LD processing if your application already knows the contexts used in the VC -- even if your library/processor supports JSON-LD. You'd only do JSON-LD processing, for example, to convert from an unknown context to a known one. In other words, library/processors that support JSON-LD will have this additional capability and they can use it as expected when necessary. I think the original text captured that part a bit better (while it was lacking on addressing the ordering issue).

Perhaps we want to say:

```
All libraries or processors MUST ensure that the order of the values in the
<code>@context</code> <a>property</a> is what is expected for the particular
application. Libraries or processors that support JSON-LD can process
the <code>@context</code> <a>property</a> using full JSON-LD processing
as expected.
```